lite/core/op_registry.cc

```diff
@@ -83,39 +83,61 @@ std::list<std::unique_ptr<KernelBase>> KernelRegistry::Create(
     case TARGET(kHost): {
```

Comment:
`LITE_ON_MODEL_OPTIMIZE_TOOL` or `LITE_WITH_PYTHON` should be considered. When we compile `opt` or `python opt ` all these backends may be needed. 
Reference: https://github.com/PaddlePaddle/Paddle-Lite/blob/0d10c60e68b9268faae73930c96a49173f24c915/lite/kernels/cuda/CMakeLists.txt#L1